aea/cli/common.py

```diff
@@ -21,36 +21,31 @@
 import logging
 import logging.config
 import os
-import pprint
 import re
 import shutil
 import sys
 from collections import OrderedDict
 from functools import update_wrapper
 from pathlib import Path
-from typing import Collection, Dict, List, Optional, cast
+from typing import Dict, List, Optional, cast
 
 import click
 
-from dotenv import load_dotenv
-
 import jsonschema  # type: ignore
 from jsonschema import ValidationError
 
-from packaging.version import Version
-
 import yaml
 
-import aea
 from aea import AEA_DIR
 from aea.cli.loggers import default_logging_config
 from aea.configurations.base import (
     AgentConfig,
     ConfigurationType,
     DEFAULT_AEA_CONFIG_FILE,
     Dependencies,
-    PackageConfiguration,
     PublicId,
+    _check_aea_version,
+    _compare_fingerprints,
```

Comment:
they have been moved to `aea.configurations.base` since now this functionality is implemented in the configuration objects.